index.js

```diff
@@ -1 +1,66 @@
 console.log("Hello!!!");
+
+//Closure
+// Create a calculate() function that takes an initial number as an argument
+// and returns methods add(), subtract(), multiply(), divide() and reset()
+// each of which takes a number as an argument and returns
+// the result of the corresponding arithmetic operation on numbers
+
+const objClosure = {
+    initialNumber: 0,
+    saveNumber: 0,
+    add(addNumber){
+        this.initialNumber += addNumber;
+        return this.initialNumber;
+    },
+    subtract(subtractNumber){
+        this.initialNumber -= subtractNumber;
+        return this.initialNumber;
+    },
+    multiply(multiplyNumber){
+        this.initialNumber *= multiplyNumber;
+        return this.initialNumber;
+    },
+    divide(divideNumber){
+        this.initialNumber /= divideNumber;
+        return this.initialNumber;
+    },
+    reset(){
+        this.initialNumber = this.saveNumber;
+    },
+};
+
+const calculate = (initialNumber) => {
+    objClosure.initialNumber = initialNumber;
```

Comment:
this is not a closure
![image](https://user-images.githubusercontent.com/32968019/225929164-908ab35c-4f8d-44fe-8891-b9a934acc810.png)